tests/test_utils.py

```diff
@@ -403,3 +539,11 @@ def test_run_planning(domain_file, problem_file, impossible_problem_file):
     with pytest.raises(NotImplementedError) as e:
         utils.run_planning(task, planner="not a real planner")
     assert "Unrecognized planner" in str(e)
+
+
+if __name__ == "__main__": # pragma: no cover
```

Comment:
instead of this, you can test this file alone by doing `pytest -s tests/test_utils.py`. and if you want to test just a single one, you can do `pytest -s tests/test_utils.py -k test_embed_training_tasks` for example